.vscode/settings.json

```diff
@@ -3,6 +3,7 @@
   "rust-analyzer.linkedProjects": ["./core/Cargo.toml"],
 
   "editor.codeActionsOnSave": {
-    "source.organizeImports": true
+    "source.organizeImports": false,
```

Comment:
Now that we rely on `eslint` for sorting, we don't want the VSCode one because they clash (source.fixAll right under does it automatically as well via eslint) cc @spolu @lasryaric